schemas/assetCostume.schema.json

```diff
@@ -249,6 +79,36 @@
               }
             },
             "allOf": [
+              { "$ref": "commonMaterial.schema.json" },
+              {
+                "if": {
+                  "properties": {
+                    "name": {
+                      "const": "Wolf3D_Body"
+                    }
+                  }
+                },
+                "then": {
+                  "properties": {
+                    "textures": {
+                      "$ref": "commonMaterial.schema.json#/$defs/normalOnlyTextureChannels"
+                    },
+                    "doubleSided": {
+                      "$ref": "commonMaterial.schema.json#/$defs/singleSided"
+                    }
+                  }
+                },
+                "else": {
+                  "properties": {
+                    "textures": {
+                      "$ref": "commonMaterial.schema.json#/$defs/allTextureChannels"
+                    },
+                    "doubleSided": {
+                      "$ref": "commonMaterial.schema.json#/$defs/anySided"
+                    }
+                  }
+                }
+              },
```

Comment:
This is wrong! Creators shall have full control over the body material for costumes. This was not in the scope of the task.